### ISSUE TITLE
Sponsoring from Google?

### ISSUE THREAD

**Issue by ixkael** [AstroHackWeek/AstroHackWeek2017#3]
I might be able to get us some sponsoring from Google, via money or a hack space. Interested?

**Comment by drphilmarshall** [comment]
Yes! Andy Connolly has Google contacts too, and will be local - are you in
touch with him, or is this a completely independent lead? Thanks Boris! :-)

**Comment by ixkael** [comment]
This is an independent lead! I will ask Google and Amazon where I have contacts.

**Comment by drphilmarshall** [comment]
Cool - thanks Boris. Keep us posted!

**Comment by ixkael** [comment]
Hi, im in touch with Google - do we have different levels of sponsorship? Any particular scheme?
